### ISSUE TITLE
Add a way for extension main files to launch urls in a browser window

### ISSUE THREAD

**Issue by Sebastian-ubs** [paranext/paranext-core#1369]
**User Story**
As a extension developer, I want to be able to open a website in the user's browser so that I can send them to more information.

Example use case: Open the currently loaded website of a [website viewer](https://github.com/paranext/platform-bible-sample-extensions/pull/14) tab in the web browser: Because website viewer is a url webview, there is no other place to add such action than in the menu.

**Description**
It's possible to launch a url in the web browser by clicking on a link in a web view, but there is not way to do that from the extension host process, e.g. add a menu entry that would open a url in the browser.

`window.open` is not available and importing packages (e.g. from electron) throws
> Error: Requiring other than papi is not allowed in extensions! 

**Implementation idea**

**Testing ideas**

**Comment by tjcouch-sil** [comment]
We can hook up Electron's `shell.openExternal` https://www.electronjs.org/docs/latest/api/shell#shellopenexternalurl-options

**Comment by Sebastian-ubs** [comment]
And with saying "hook up" you mean exposing this from the papi, because only papi methods can be imported?

**Comment by tjcouch-sil** [comment]
Yes, I am putting it in a command `platform.openWindow` in https://paratextstudio.atlassian.net/browse/PT-1640
